src/multiParamManager.js

```diff
@@ -77,13 +76,13 @@ class MultiParamManager {
       this.clickCallBack(this.funABI.inputs, ret)
     }
 
-    var button = yo`<button onclick=${() => { multiOnClick() }} class="${css.instanceButton}">Submit</button>`
+    var button = yo`<button onclick=${() => { multiOnClick() }} class="${css.instanceButton}">Call</button>`
```

Comment:
Not sure it is a good idea to rename it `Call`, that could add confusion cause we also use the word `Call` when executing a function without changing the state (in remix blue action versus red action). maybe `Transact`?

Reply:
True!  Probably Transact or Snd Txn - I'll make it Transact for now.